### PR TITLE
Added region to the list response

### LINKED ISSUE #15: Add Region to the list output
Right now the [list](https://github.com/nodeschool/admin/blob/master/lib/standard/list.js) method only returns a array of chapters but since the [`chapter-designation.js`](https://github.com/nodeschool/admin/blob/master/country_designations.js) file exists it would be good if the area was automatically added to each item in the list.

A PR that fills the json resulting from `list` with an area lookup would be mighty welcome.

### PR DESCRIPTION
**Pull request by dinodsaurus**
closes #15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nodeschool-admin",
-  "version": "0.1.0",
+  "version": "0.1.2",
```
Comment (martinheidegger) [inline package.json]: The version shouldn't change in a PR.

File: lib/util/validate.js [hunk 2]
```diff
@@ -63,5 +64,21 @@ module.exports = {
   },
   chapter: function (path, data) {
     return this.validate(require('./schema/chapter'), path, data)
+    .then(function (chapter) {
+      if (!countries) {
+        countries = require('../../country_designations')
+      }
+      var cntrs = countries.byGroup
```
Comment (martinheidegger) [inline lib/util/validate.js]: Did you see the `byCountry` method at the end of `country_designations`?
https://github.com/nodeschool/admin/blob/master/country_designations.js#L271

It seems you have double-implemented it?!

File: package.json (hunk 1 above)
Comment (dinodsaurus) [inline package.json]: makes sense :+1:

File: lib/util/validate.js (hunk 2 above)
Comment (dinodsaurus) [inline lib/util/validate.js]: sorry, totally missed it out

**Comment by dinodsaurus** [comment]
I think we can merge it now, what do you think @martinheidegger

**Comment by martinheidegger** [comment]
Yes, looks good to me

**Pull request merged**